whisper/whisperv6/doc.go

```diff
@@ -36,15 +36,15 @@ import (
 
 const (
 	EnvelopeVersion    = uint64(0)
-	ProtocolVersion    = uint64(5)
-	ProtocolVersionStr = "5.0"
+	ProtocolVersion    = uint64(6)
+	ProtocolVersionStr = "6.0"
 	ProtocolName       = "shh"
 
 	statusCode           = 0 // used by whisper protocol
 	messagesCode         = 1 // normal whisper message
 	p2pCode              = 2 // peer-to-peer message (to be consumed by the peer, but not forwarded any further)
 	p2pRequestCode       = 3 // peer-to-peer message, used by Dapp protocol
-	NumberOfMessageCodes = 64
+	NumberOfMessageCodes = 128
```

Comment:
Why did you make this change?

Reply:
Because the [EIP-627 packet codes](https://github.com/gluk256/EIPs/blob/effb874963b6a0edb7b26c60db0eaf5df9bf8e56/EIPS/eip-draft-whisper.md#packet-codes) specifies that:

```
The message codes reserved for Whisper protocol: 0 - 127. Messages with unknown codes must be ignored, for forward compatibility of future versions.
```

Support for messages with codes 126 and 127 will be added in subsequent PRs.

Reply:
This change to EIP was made after discussion with Robert (Parity) several months ago.